src/app-logic/url-handling.js

```diff
@@ -154,7 +155,7 @@ type FullProfileSpecificBaseQuery = {|
   // value can't be upgraded using the typical URL upgrading process, as the full profile
   // must be fetched to compute the tracks.
   threadOrder: string, // "3-2-0-1"
-  hiddenThreads: string, // "0-1"
+  hiddenThreads: string, // "0-1",
```

Comment:
nit: it looks like there's an extraneous comma at the end, can you please remove it?